<?xml version="1.0" ?>
<launch>
    <arg name="markerSize" default="0.04" />
    <include file="$(find panda_cartesian_impedance)/launch/tracking_system.launch">
        <arg name="markerSize" value="$(arg markerSize)" />
    </include>
    <!-- read object pose from marker, and command the arm !-->
    <node name="debug" pkg="panda_cartesian_impedance" type="debug_multi_marker_obs.py" required="true" output="screen">
        <!--remap from="equilibrium_pose" to="/cartesian_impedance_example_controller/equilibrium_pose" /!-->
        <param name="markers_topic" value="marker_publisher/markers" />
    </node>
</launch>